src/ProjectOrigin.WalletSystem.IntegrationTests/TestClassFixtures/RegistryFixture.cs

```diff
@@ -44,6 +50,18 @@ public RegistryFixture()
             .WithName(Guid.NewGuid().ToString())
             .Build();
 
+        _rabbitMqImage = new ImageFromDockerfileBuilder()
+            .WithDockerfileDirectory(CommonDirectoryPath.GetProjectDirectory(), string.Empty)
+            .WithDockerfile("rabbitmq.dockerfile")
+            .Build();
```

Comment:
Why this exact image?

Reply:
It is to enable  rabbitmq_management which is required  by the registry.